fabric-client-gametest-api-v1/src/client/resources/fabric-client-gametest-api-v1.mixins.json

```diff
@@ -0,0 +1,25 @@
+{
+  "required": true,
+  "package": "net.fabricmc.fabric.mixin.client.gametest",
+  "compatibilityLevel": "JAVA_21",
+  "mixins": [
```

Comment:
The split of `mixins` and `client` is pointless here, as this whole module is client only. Id just put them all in `mixins`

Reply:
That's mcdev's fault, I don't manually edit these :D